.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python: 3.5
+
```

Comment:
Could you add `cache: pip` so that [Travis caches dependencies](https://docs.travis-ci.com/user/caching/#pip-cache)?